server/routes/viewingLicences/handlers/printLicence.ts

```diff
@@ -178,7 +178,7 @@ export default class PrintLicenceRoutes {
     const licenceKind = licence.kind
     const licenceType = licence.typeCode
     if (licenceKind === LicenceKind.HDC) {
-      return 'HDC_AP'
+      return 'HDC_AP_PSS'
```

Comment:
Changed temporarily to enable testing in dev - next ticket covers logic to show `HDC_AP` or `HDC_AP_PSS` depending on type.